src/profiler/profilerthread.cpp

```diff
@@ -40,15 +41,19 @@ Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 
 // DE: 20090325: Profiler has a list of threads to profile
 // RM: 20130614: Profiler time can now be limited (-1 = until cancelled)
-ProfilerThread::ProfilerThread(HANDLE target_process_, const std::vector<HANDLE>& target_threads, SymbolInfo *sym_info_)
+ProfilerThread::ProfilerThread(HANDLE target_process_, const std::vector<std::pair<HANDLE, DWORD>>& target_threads, SymbolInfo *sym_info_)

```

Comment:
Would it be simpler to use `GetThreadId` instead of additionally passing the ID around?